README.md

```diff
@@ -47,6 +47,25 @@ All [configuration variables](https://github.com/eccenca/gui-elements/blob/devel
 
 ## Development
 
+### Branch management
+
+We have 4 types of major branches representing the current state:
+
+-   `main`: contains the latest official release, only release branches will be merged in this branch
+-   `develop`: contains the latest state of development, all feature and bugfix branches and here via PRs
```

Comment:
Something seems to be missing in  `all feature and bugfix branches and here via PRs`